src/App/Components/Login/Login.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import SocialLogin from '../SocialLogin/socialLogin';
 
 const Login = () => (
   <React.Fragment>
     <h1>Authors Heaven/Login</h1>
     <p>Thi is the login page</p>
+    <SocialLogin />
```

Comment:
Why do we have this file? Is being used anywhere

Reply:
which file specifically?